### PR TITLE
ivnas post about reproducibility

### PR DESCRIPTION
**Pull request by iivankovic**
A quest for reproducible data science!

### CODE REVIEW

File: content/post/about-reproducibility/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: A Quest for Reproducible Data Science
+summary: Have you ever tried to reproduce someone else’s analysis and got different results? 
+authors: 
+- ivnaivankovic
+date: 2020-09-22
+tags: 
+- Reproducibility, Data Science
```
Comment (lokijuhy) [inline content/post/about-reproducibility/index.md:8]: This is making one tag called "Reproducibility, Data Science". Put them in separate lines to make 2 tags.

**Review by lokijuhy — CHANGES_REQUESTED** [review]
Love it! I went to see if you added a "reproducibility" tag, so we can make a series of these articles and have them be connected, and was delighted to see that you did! :) There's one tiny syntax error to make the tag work properly though.

File: content/post/about-reproducibility/index.md (hunk 1 above)
Comment (iivankovic) [reply content/post/about-reproducibility/index.md]: Nice catch, fixed!

**Review by lokijuhy — APPROVED** [review]
💯

**Pull request merged**